configure.ac

```diff
@@ -28,11 +28,11 @@ AC_ARG_ENABLE([profile], AS_HELP_STRING([--enable-profile],
 AC_ARG_ENABLE([fuzz], AS_HELP_STRING([--enable-fuzz],
     [enable fuzzing consumer @<:@default=no@:>@]))
 AC_ARG_ENABLE([fontconfig], AS_HELP_STRING([--disable-fontconfig],
-    [disable fontconfig support @<:@default=check@:>@]))
+    [disable Fontconfig support @<:@default=check@:>@]))
 AC_ARG_ENABLE([directwrite], AS_HELP_STRING([--disable-directwrite],
     [disable DirectWrite support (Windows only) @<:@default=check@:>@]))
 AC_ARG_ENABLE([coretext], AS_HELP_STRING([--disable-coretext],
-    [disable CoreText support (OSX only) @<:@default=check@:>@]))
+    [disable Core Text support (macOS only) @<:@default=check@:>@]))
```

Comment:
In fact, this isn’t actually correct: Core Text is also available on iOS and probably on Apple’s other platforms (watchOS, tvOS…) if libass can be built there at all. Should we change this to “macOS/iOS”? Or something else?

Reply:
I usually just say "Apple" (ie "(Apple only)" here).

Reply:
Added a commit that switches to “Apple”.